plugins/org.eclipse.n4js/src/org/eclipse/n4js/validation/messages.properties

```diff
@@ -553,6 +553,8 @@ VCO_IDENT_ILLEGAL_CHAR_WITH_RESULT=error;;;Illegal character in identifier ''{0}
 # 0: identifier, 1: position
 VCO_IDENT_ILLEGAL_CHAR=error;;;Illegal character in identifier ''{0}'' at position {1}.
 # no parameters required
+VCO_JSXIDENT_WHITESPACE_COMMENT=error;;;JSX attribute names may not contain whitespace or comments.
```

Comment:
Can you note in the error message that attribute names ending with '-' and directly followed by another attribute name are merged and must not contain whitespace or comments.

Reply:
What would be a good wording?
`SX attribute names may not contain whitespace or comments. Attribute names ending with a '-' and directly followed by another attribute name are merged and must not contain whitespace or comments.` ?

Reply:
👍